### DISCUSSION TITLE
输入法通用词库标准

### DISCUSSION THREAD

**Discussion by xingwozhonghua126** [fcitx/fcitx5#353] [Ideas]
只要是文本的就行，每行包含:词条，编码，词频。

![mmexport1633969658983](https://user-images.githubusercontent.com/18661883/136824200-6b4caeba-2a21-4d47-917e-01340713f8f6.jpg)
![mmexport1633969654612](https://user-images.githubusercontent.com/18661883/136824207-2755c3a7-f80a-4a6a-b01f-d7b569ce0b42.jpg)

建议通用词库格式支持主流开源输入法引擎导入导出。

**Comment by CoelacanthusHex** [comment]
问题在于，每个输入法采用的词频可能完全不同，之间的转换是难以（转换公式过于复杂）甚至不可能（设计思路完全相悖）完成的

**Reply by xingwozhonghua126** [reply]
是的,
我的想法是我们先设立一个标准对于以后新开发的输入法都符合这个标准就可以了嘛,
之前的就不考虑了,能pr就pr.

当然肯定是不能所有输入法都做全的,
不过可以让大多数输入法保持相同的特性规则,
或是至少有人要说出这个事情,让大家达成共识.

或者可以为以后的linux软件生态做参考.

https://github.com/studyzy/imewlconverter
